src/controller/users/authorsGet.ts

```diff
@@ -0,0 +1,15 @@
+import { pool } from '../../DB/dbConfig';
+import { Request, Response } from 'express';
+
+export async function authorsGet(_req: Request, res: Response): Promise<void> {
+	try {
+		const authors = await pool.query(`SELECT * FROM users WHERE is_author = $1`, ['true']);
```

Comment:
Also I would recommend to move queries to separate file. Then you can reuse the same query in different places